src/content/developers/docs/consensus-mechanisms/pow/mining-algorithms/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Mining algorithms
+description: A detailed look at the algorithms used for Ethereum mining.
+lang: en
+sidebar: true
+incomplete: false
+---
+
+Ethereum mining has used two mining algorithms, Dagger Hashamoto and Ethash. Dagger Hashamoto is no longer used to mine Ethereum, being superseded by Ethash. However, it has historical significance as an important innovation in Ethereum's development. Proof-of-work mining itself will be deprecated in favor of proof-of-stake during [The Merge](/merge/), which is forecast to happen in Q3-Q4 2022. 
+
+The fundamental idea of both algorithms is that a miner tries to find a nonce input using brute force computation so that the result is below a certain difficulty threshold. This difficulty threshold can be dynamically adjusted, allowing block production to happen at a regular tempo.
```

Comment:
```suggestion
The fundamental idea of both algorithms is that a miner tries to find a nonce input using brute force computation so that the result is below a certain difficulty threshold. This difficulty threshold can be dynamically adjusted, allowing block production to happen at a regular interval.
```